src/sections/AddressInfoSection.tsx

```diff
@@ -115,40 +134,38 @@ const TransactionInfoSection = () => {
     <Section>
       <SectionTitle title="Address" isLoading={infoLoading || txLoading} />
 
-      {!infoLoading && !addressInfo ? (
-        <InlineErrorMessage message={addressInfoError} />
-      ) : (
-        <Table bodyOnly isLoading={infoLoading} minHeight={250}>
-          {addressInfo && (
-            <TableBody tdStyles={AddressTableBodyCustomStyles}>
-              <TableRow>
-                <td>Address</td>
-                <HighlightedCell textToCopy={id} qrCodeContent={id}>
-                  {id}
-                </HighlightedCell>
-              </TableRow>
-              <TableRow>
-                <td>Number of Transactions</td>
-                <td>{addressInfo.txNumber}</td>
-              </TableRow>
-              <TableRow>
-                <td>Total Balance</td>
-                <td>
-                  <Badge type={'neutralHighlight'} amount={addressInfo.balance} />
-                </td>
-              </TableRow>
-              {addressInfo.lockedBalance && parseInt(addressInfo.lockedBalance) > 0 && (
-                <TableRow>
-                  <td>Locked Balance</td>
-                  <td>
-                    <Badge type={'neutral'} amount={addressInfo.lockedBalance} />
-                  </td>
-                </TableRow>
+      <Table bodyOnly minHeight={250}>
+        <TableBody tdStyles={AddressTableBodyCustomStyles}>
+          <TableRow>
+            <td>Address</td>
+            <HighlightedCell textToCopy={id} qrCodeContent={id}>
+              {id}
+            </HighlightedCell>
+          </TableRow>
+          <TableRow>
+            <td>Number of Transactions</td>
+            <td>{txNumber ?? <LoadingSpinner size={14} />}</td>
```

Comment:
As shown in the video, maybe we want to stop spinning when an error occurs? Instead showing an error message (in italic, secondary color?), as we do for the table below?

Reply:
ok, on it!

Reply:
Better? (btw, I am using the updated SDK for displaying the error messages https://github.com/alephium/js-sdk/pull/99)

<img width="1134" alt="image" src="https://user-images.githubusercontent.com/1579899/162178471-942a626d-e9d7-4682-a530-723c9f40f1c3.png">

Reply:
lol, just noticed my typos

Reply:
Perfect.